op-deployer/pkg/deployer/report/cli.go

```diff
@@ -0,0 +1,82 @@
+package report
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer"
+	"github.com/ethereum-optimism/optimism/op-service/ioutil"
+	"github.com/ethereum-optimism/optimism/op-service/jsonutil"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/rpc"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	DeploymentTxHashFlag = &cli.StringFlag{
+		Name:    "deployment-tx-hash",
+		Usage:   "The transaction hash of the deployment.",
+		EnvVars: deployer.PrefixEnvVar("DEPLOYMENT_TX_HASH"),
+	}
+	ReleaseFlag = &cli.StringFlag{
+		Name:    "release",
+		Usage:   "The release tag of the deployment, of the format 'op-contracts/vX.Y.Z'.",
+		EnvVars: deployer.PrefixEnvVar("RELEASE"),
+	}
+)
+
+var L1ReportFlags = []cli.Flag{
+	deployer.L1RPCURLFlag,
+	deployer.OutfileFlag,
+	DeploymentTxHashFlag,
+	ReleaseFlag,
+}
+
+var Commands = cli.Commands{
+	{
+		Name:   "l1",
+		Usage:  "Generates a report for a chain whose L1 contracts were deployed using op-deployer.",
+		Flags:  L1ReportFlags,
+		Action: ScanL1CLI,
+	},
```

Comment:
Specifying L1 report here implies there will be an option for L2 report as well. Do you think most times a user will want one versus the other, or is it more likely they will run both simultaneously to get a "full" report?